### PR TITLE
De running a server

### PR DESCRIPTION
**Pull request by reinhardwh**
**DE Translation for Running-a-Server**

**DE Translation for Running-a-Server**

**Translation ok**

**Proof Reading**

**Its a translation**

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

**Comment by reinhardwh** [comment]
Hope all good

**Comment by ann0see** [comment]
Will look at it as soon as possible! Thank you very much by the way. Just to clarify: I'd focus on the most used sites (running a server is an edge case but good to have translated anyway ;-)).
@ignotus666 knows a lot about the translation process, so I think it would be good to have his review too.

File: _translator-files/po/de/Running-a-Server.po [hunk 1]
```diff
@@ -45,283 +47,316 @@ msgid ""
...
+msgstr "## Muss ich einen Server betreiben?"
 
 #. type: Plain text
-#: ../wiki/en/Running-a-Server.md:24
-#, no-wrap
-msgid "**No**. You can use the Servers listed by the built in Directories and use Jamulus without running a Server or choose a third party hosting service. If you just want an undisturbed session, use the [soloing technique described on the Tips and Tricks page](Tips-Tricks-More#have-a-undisturbed-session-on-any-server). If you decide you cannot use any of the Servers listed by the built in Directories, you may be able to use a Server (either Registered in a Custom Directory, or Unregistered - see [Server Types](#server-types)) hosted by a third party. Doing so will save you the trouble of setting one up yourself.\n"
-msgstr ""
+#: ../wiki/en/Running-a-Server.md:25
+msgid "The short answer is **no**. For various reasons, you should try to use other people's servers, at least at first while getting your sound set up with the Jamulus client."
+msgstr "Die kurze Antwort lautet **nein**. Aus verschiedenen Gründen sollten Sie versuchen, die Server anderer Leute zu benutzen, zumindest am Anfang, während Sie Ihren Sound mit dem Jamulus-Client einrichten."
```
Comment (ann0see) [inline _translator-files/po/de/Running-a-Server.po:86]: I think we're using du on other pages

**Comment by reinhardwh** [comment]
If you like please share a prio list for most used pages.
Then i would follow this route.

**Review by ann0see — COMMENTED** [review]
From a first quick glance it looks good. Only the fact that we're using informal du is probably a problem

File: _translator-files/po/de/Running-a-Server.po (hunk 1 above)
Comment (reinhardwh) [reply _translator-files/po/de/Running-a-Server.po:86]: Ok thats easy to fix. 
Do you prefer more formal "Sie" or the friendly "Du"

**Comment by ann0see** [comment]
I think it's something like

1. Startpage
2. Getting Started page
3. Installation for Win, mac, Linux

now a bit less important 

4. Running a server
5. Client manual
6. Installation for iOS/Android
Afterwards Trobleshooting probably?

**Comment by ignotus666** [comment]
Hi @reinhardwh, thanks for contributing!

You seem to be translating the .po files from the `release` branch. You need to translate those from `next-release` instead. To create a branch based on `next-release`, first select it (next-release) in the branches drop-down menu, and then create your branch based on it by filling in the "Find or create a branch" field.

**Comment by reinhardwh** [comment]
Ok, sorry
I will follow the steps and create a fresh pull tomorrow, ok?

**Comment by ignotus666** [comment]
Yes that's fine. If you followed the step in `_translator-files/README.md` to create a translation memory for Poedit you will be able to reuse most of what you already translated instead of having to do it all over again.

**Comment by reinhardwh** [comment]
> Hi @reinhardwh, thanks for contributing!
> 
> You seem to be translating the .po files from the `release` branch. You need to translate those from `next-release` instead. To create a branch based on `next-release`, first select it (next-release) in the branches drop-down menu, and then create your branch based on it by filling in the "Find or create a branch" field.

Translation memory working fine, thanks!

One potential issue:
There is a MD file include in Backup Session. This will create a mix of DE and EN most likely.
grafik
Should I translate a DE MD file for this or better to have a PO file -right?

**Comment by ignotus666** [comment]
There's a .po file in your language folder called "Include-Backing-Up.po" - you would translate that.

**Comment by reinhardwh** [comment]
ok, then i will change the link in DE translation
grafik

**Comment by ignotus666** [comment]
No, no - leave it linking to the .md file. All .po files are automatically converted into .md files by a Github action, you don't need to worry about that.

**Comment by ignotus666** [comment]
You should leave that link as-is: `{% include_relative Include-Backing-Up.md %}`. gh-pages will choose the translated DE .md file without having to explicitly tell it to.

**Comment by ignotus666** [comment]
Closing this PR as it's being replaced by a new one.

**Comment by reinhardwh** [comment]
@ignotus666 shall I delete the "branch"?

**Comment by ignotus666** [comment]
You can if you want to.

**Comment by reinhardwh** [comment]
Would this delete the comments as well?
If yes - I would like to keep the branch for reference

**Comment by ignotus666** [comment]
Don't worry, the comments will stay here.

**Pull request closed without merging**